src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3997,6 +4121,130 @@ namespace ZoneTempPredictorCorrector {
 				SNLoad = 0.0;
 			}
 
+			//Hybrid modeling start: Added by Sang Hoon Lee May 2015
+			if (( HybridModelZone( ZoneNum ).InternalThermalMassCalc || HybridModelZone( ZoneNum ).InfiltrationCalc ) && !WarmupFlag && !DoingSizing ) {
+
+				Zone( ZoneNum ).ZoneMeasuredTemperature = GetCurrentScheduleValue( HybridModelZone( ZoneNum ).ZoneMeasuredTemperatureSchedulePtr );
+
+				// prepare start and end date for Hybrid Modeling
```

Comment:
Looks like HybridStartDayOfYear and HybridEndDayOfYear are being recomputed every timestep?  These should be done in an init function or just added to the hybridzone getinput and stored in the HybridModelZone struct.